pulp_container/app/tasks/sync_stages.py

```diff
@@ -349,7 +351,6 @@ def create_tagged_manifest(self, tag_name, saved_artifact, manifest_data, raw_da
             raw_data: (str): The raw JSON representation of the ImageManifest.
 
         """
-        media_type = manifest_data.get("mediaType", MEDIA_TYPE.MANIFEST_V1)
```

Comment:
for the case when `mediatype` is not available you should decide what kind of manifest is this based on the `config` media type. `Config` is a required field, same as `mediatype` for it.

Reply:
I forgot about schema1 :/ I believe if config is not available then we can assume this is a docker schema1, wdyt?

Reply:
Exactly. This is the reason why I left the media_type field unchanged.

Reply:
yes so the logic would be:
```
if not media_type:
    if config =: manifest_data.get("config"):
        config_media_type = config. get("media_type")
        if config_media_type == CONFIG_BLOB_OCI:
            media_type = schema2_image_oci
       else:
           media_type = schema2_image_docker
    else:
        media_type = schema1
```

Reply:
I have defined a new function for determining the schema in utils.py.